superset/assets/javascripts/explore/main.css

```diff
@@ -15,6 +15,51 @@
   margin-right: 0px;
 }
 
+.table-container {
```

Comment:
should be in `nvd3.css` and the css should be namespaced to only affect what is under `.nvd3`